core-bundle/contao/templates/twig/backend/template_studio/editor/_editor_tab.html.twig

```diff
@@ -57,7 +60,40 @@
             {# Relation to next representation #}
             {% if not loop.last %}
                 <div class="relation">
-                    {{ backend_icon('arrow-down') }}
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"
+                         fill="none"
+                         stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                        <path d="M8 18L12 22L16 18"/>
+                        <path d="M12 2V22"/>
+                    </svg>
```

Comment:
Please don't use inline SVGs. They bloat the source code and we cannot generate dark mode variants for them.

Reply:
Alternatively they could also be set via CSS via mask-image. Then you do not need two files.